.github/workflows/ci.yml

```
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        php: ['8.1', '8.2']
+        php: ['8.2']
```

We may keep it ready for 8.3:
```suggestion
        php: ['8.2', '8.3', 'nightly']
```